static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -383,3 +396,98 @@ $pillars: (
         color: #ffffff;
     }
 }
+
+
+.fc-item--pillar-special-report-alt,
+.fc-item--pillar-special-report-alt.fc-item--type-feature {
+    &.fc-item--type-immersive.fc-item--has-boosted-title.fc-item--standard-mobile {
+        .fc-item__content {
+            background-color: $special-report-alt-faded;
+        }
+
+        &:hover {
+            .fc-item__content {
+                background-color: $special-report-alt-faded;
+            }
+        }
+    }
+
+    background-color: $special-report-alt-faded !important;
+
+    .fc-item__container.u-faux-block-link--hover {
+        background-color: $special-report-alt-faded !important;
```

Comment:
I am sorry I had to add `!important` to so many places but it wouldn't pick up the right colour for SpecialReportAlt. Taking into account that fronts migration to DCR is in progress I don't think this is a huge problem but happy to discuss if anyone thinks otherwise!

Reply:
Agreed!